go/ct/spc/specification.go

```diff
@@ -1455,6 +1455,89 @@ func getAllRules() []Rule {
 		},
 	})...)
 
+	// --- CREATE2 ---
+
+	rules = append(rules, rulesFor(instruction{
+		op:        CREATE2,
+		name:      "_staticcall",
```

Comment:
The term `staticcall` might be misleading since there is a `STATICCALL` instruction. How about just "_static"?